### ISSUE TITLE
Please list altered lighting variables

### ISSUE THREAD

**Issue by thedanish** [RobertBeckebans/RBDOOM-3-BFG#410]
Clearly things have changed since the original Doom3BFG as the light variables have drastically different effects.

For example using:
set r_lightscale 1
set r_overbright 1
used to bring the overly bright BFG lighting back down to acceptable Doom3 levels, now it appears to do nothing.

In addition, settings from mods such as Ultimate HD (http://www.moddb.com/mods/doom-3-bfg-ultimate):
set r_hdrKey "0.3"
set r_hdrContrastOffset "3000"
set r_hdrContrastThreshold "16"
set r_useFilmicPostProcessEffects "0" //removes blue shift
set r_forceAmbient "0.0025" //reduces ambient illumination
set r_lightScale "2.5"
set r_gamma "0.85"
result in eye shattering brightness.

It would be of great value if you could list which lighting variables have been altered for users to adjust accordingly. If this is already specified somewhere then adding a link to that in the Readme would greatly assist dumdums like myself. Thank you.

**Comment by y2keeth** [comment]
those are all rbdoom3bfg settings

**Comment by thedanish** [comment]
y2keeth, yes they are but the values controlling them or the effects they have has been changed. For example the Ultimate HD mod I linked uses an older version of rbdoom3bfg, those settings work correctly on the older version but not on the current version. I know that specifically because I used to use an old version too and it worked, then one day I updated and the lightscale/overbright settings no longer worked and on the current version the mod settings also no longer work. Attached is the Ultimate HD mod with the version I cloned and compiled yesterday, brightness turned all the way down in settings menu.
![screenshot at 2018-01-12 13-32-47](https://user-images.githubusercontent.com/11960929/34877337-9245338a-f79d-11e7-9344-dbd490a15763.png)
![screenshot at 2018-01-12 13-33-38](https://user-images.githubusercontent.com/11960929/34877428-e9482368-f79d-11e7-982d-dbd22804fa65.png)

I'm not demanding anything be changed back, I simply want to know which settings have been changed.
